### DISCUSSION TITLE
Different color for the same file type. Why?

### DISCUSSION THREAD

**Discussion by ralph-bergmann** [eza-community/eza#828] [Q&A, Howto, Help] [answered]
I installed it today, and it looks good so far, but I don't understand the meaning of the file name colors and why they are different.

<img width="820" alt="SCR-20240208-osgo" src="https://github.com/eza-community/eza/assets/1191649/ed9631f6-b9c2-4b7c-8f53-b840b9aa0a42">

Why does `README-PAY.png` have a different color than `architecture.png` or `data_sync_strategy.jpg`?
The same with `README-PAY.drawio`. Why has it a different color than `checkin_tan_overview.drawio`?

**Comment by Delapouite** [comment, marked as answer]
Hi. It's because of the `README` prefix: https://github.com/eza-community/eza/blob/708f7be887055ff45d7849d6d730bdecc920822f/src/info/filetype.rs#L385
